### PR TITLE
REPO-3721: Admin console link

### PR DESCRIPTION
**Pull request by Epurashu**
docker-compose : Add external alfresco.host and alfresco.port to JAVA_OPTS for repository image in order to fix AOS
   docker-compose : Add external aflresco.host alfresco.port alfresco.context and alfresco.protocol to JAVA_OPTS for share image in order to fix broken admin console link
   helm : Add CATALINA_OPTS to config-share. Add alfresco.proxy value to fix broken admin console link

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -76,12 +78,19 @@ services:
             - 8099:8099
 
     share:
-        image: alfresco/alfresco-share:6.0
+        image: alfresco-share-test
```
Comment (alandavis) [inline docker-compose/docker-compose.yml]: Looks like we are going to have to wait for a Share release.
Might be better to not change the Share version and to get the rest of the changes merged to master so they will be picked up when there is a Share release.

File: docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -76,12 +78,19 @@ services:
...
-            - "CATALINA_OPTS= -Xms500m -Xmx500m"
+            REPO_HOST: "alfresco"
+            REPO_PORT: "8080"
+            JAVA_OPTS: "
+                -Xms500m
+                -Xmx500m
+                -Dalfresco.host=localhost
+                -Dalfresco.port=8082
+                -Dalfresco.context=alfresco
+                -Dalfresco.protocol=http
```
Comment (alandavis) [inline docker-compose/docker-compose.yml]: Is this correct? Would it normally be https externally?

Comment (Epurashu) [reply docker-compose/docker-compose.yml]: We are currently not using HTTPS for our docker-compose deployment.

File: docker-compose/docker-compose.yml (hunk 1 above)
Comment (Epurashu) [reply docker-compose/docker-compose.yml]: This one slipped into the commit, I`m going to revert this change.

**Pull request merged**